platform/server/http/HttpService.ts

```diff
@@ -23,38 +23,29 @@ export class HttpService implements CoreService {
   }
 
   async start() {
-    this.configSubscription = this.config$
-      .filter(() => {
-        if (this.httpServer.isListening()) {
-          // If the server is already running we can't make any config changes
-          // to it, so we warn and don't allow the config to pass through.
-          this.log.error(
-            'Received new HTTP config after server was started. ' +
-              'Config will **not** be applied.'
-          );
-          return false;
-        }
+    this.configSubscription = this.config$.subscribe(() => {
+      if (this.httpServer.isListening()) {
+        // If the server is already running we can't make any config changes
+        // to it, so we warn and don't allow the config to pass through.
+        this.log.warn(
```

Comment:
It feels like `warn` is more appropriate here since Kibana will still function normally.

Reply:
Yeah, I see your point. For now this is good, though.